### ISSUE TITLE
GCop420 false alarm

### ISSUE THREAD

**Issue by sungam3r** [Geeksltd/GCop#285]
```csharp
 public class XmlGenericSerializer 
    {
        private readonly XmlSchemaSet _schemaSet;

        public XmlGenericSerializer(XmlSchemaSet schemaSet) 
        {
              _schemaSet = schemaSet;
              // some logic
        }

        public XmlGenericSerializer() // GCop420 Constructor should not be empty. 
        {
        }
    }
```
I can't just erase the default constructor, because by doing so I will oblige clients to pass unnecessary parameters.

It seems to me that you need to separate the rule for empty methods and for default constructors.

**Comment by sungam3r** [comment]
By the way, if you put a comment in the body of the default constructor, the warning disappears.

```csharp 
public XmlGenericSerializer()
        {
            // GCop420
        }
```

**Comment by marjanjavid** [comment]
@Karvan Should I change this rule or we should solve this problem by comments?

**Comment by Karvan** [comment]
we need to keep the rule. Devs can skip it or add a comment. no need to update the logic or documentation.
